tests/unit/package_managers/test_gomod.py

```diff
@@ -339,12 +336,7 @@ def test_resolve_gomod_vendor_dependencies(
     mock_version_resolver.get_golang_version.return_value = "v0.1.0"
     mock_go_release.return_value = "go0.1.0"
     mock_get_gomod_version.return_value = ("0.1.1", "0.1.2")
-
-    flags: list[Flag] = ["gomod-vendor"]
-    if force_gomod_tidy:
-        flags.append("force-gomod-tidy")
```

Comment:
Why are we dropping the tidy flag? We're running vendor automatically, so we don't need the vendor flag, okay, but why are we dropping tidy if the CLI flag were used? The rest of the fetch tests kept the flag untouched. What am I missing?